inc/dropdown.class.php

```diff
@@ -822,274 +822,164 @@ static function getStandardDropdownItemTypes() {
       if (is_null($optgroup)) {
          $optgroup = [
              __('Common') => [
-                 'Location'               => Location::getTypeName(Session::getPluralNumber()),
```

Comment:
All entries were in facts using the getTypeName, either directly or copy/pasted. I've kept an array with null value to be compliant with existing (for plugins); doing differently would certainly require an extra foreach on all entries.